### PR TITLE
Translation key correction of wording

### PR DESCRIPTION
**Pull request by yasudacloud**
### What does it do?

Fixed the translation key for the mail plugin.

### Why is it needed?

I was working on a plugin that translates into Japanese, and I noticed that it didn't translate correctly.

### How to test it?

Run examples/getstarted to enable the Japanese language setting.
This is because the default English and French languages do not have a translation key.

When I open the page in question, the buttons and text fields that should have translation keys are still in English.

Settings > Configuration
fix_before
fix_after

### CODE REVIEW

**Review by vincentbpro — COMMENTED** [review]
Thanks for your contribution! LGTM
Could you please sign the CLA with all the email adresses you commited with (it is blocking the merge)

**Comment by yasudacloud** [comment]
@vincentbpro 

Thanks for the reply.
Regarding CLA, I have the wrong email address, what should I do?
Take down the PR once?

Sorry I'm new to this.

**Comment by vincentbpro** [comment]
@yasudacloud No worries :)
- You can change the commit author on your local branch ([example here](https://stackoverflow.com/questions/750172/how-to-change-the-author-and-committer-name-and-e-mail-of-multiple-commits-in-gi))
- Then you can force push your branch to rewrite commit history and we should be good

**Comment by yasudacloud** [comment]
@vincentbpro 

Thanks!
I tried to fix it, but what do you think?

**Comment by vincentbpro** [comment]
Thanks! should be good :)

**Review by vincentbpro — APPROVED** [review]

**Pull request merged**
